apps/src/templates/studioHomepages/BorderedCallToAction.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Radium from 'radium';
```

Comment:
Is there a way to remove the reliance on Radium in this refactor?  I don't feel strongly about it, but have a vague notion we wanted to move away from using it where possible since it might be incompatible with upgrades we'd to do. Maybe @madelynkasula knows more?

Reply:
Good question. I have no context about when we use Radium. I tried removing it and then SetUpSections wasn't happy. But I didn't explore more––would love to hear what current best practices are

Reply:
radium is deprecated, so we should remove it from any new components. it looks like the only radium feature you're using is on line 24, which allows for the `style` attribute to accept an array, where it normally requires an object (example of what i mean below). so radium should be pretty straightforward to refactor away unless i'm missing something else!

```javascript
// radium-only
<div style={[styles.outerBox, borderStyle]}></div>

// default, same result
<div style={{...styles.outerBox, ...borderStyle}}></div>
```

Reply:
Oh thank you for the example! Sounds good.